src/main/java/frc/robot/Constants.java

```diff
@@ -198,8 +198,16 @@ public static class PhotonVisionConstants {
 
   public static class IntakeConstants {
     //TODO: Assign a real CAN ID
-    public static final int kIntakeMotorID = 0;
+    public static final int kTopIntakeMotorID = 0;
+    public static final int kBottomIntakeMotorID = 1;
     public static final int kIntakeCurrentLimit = 40;
+    public static final double kShaftDiameter = 0.0127;
+    public static final double kTopGearRatio = 0.83;
+    public static final double kBottomGearRatio = 1;
+    public static final double kTopVelocityConversionFactor = (kShaftDiameter * Math.PI)/kTopGearRatio/60;
+    public static final double kBottomVelocityConversionFactor = (kShaftDiameter * Math.PI)/kBottomGearRatio/60;
```

Comment:
The only way I can readily tell these are correct is by looking at e.g. `kDriveVelocityConversionFactor` for comparison. At a minimum there should be a comment saying what units these convert between.

Reply:
I made the comment, and it is now on line 207 in constants.java